build.gradle.kts

```
@@ -15,6 +15,7 @@ dependencies {
     testImplementation("org.junit.jupiter:junit-jupiter")
     implementation(libs.flyway)
     implementation(libs.postgresql)
+    implementation(libs.testcontatiners)
```

This should be `testImplementation` because it will be used only in tests.
```suggestion
    testImplementation(libs.testcontatiners)
```